src/locales/en-gb.d.ts

```diff
@@ -0,0 +1,96 @@
+interface enGB {
+  locale: () => string;
+  actions: {
+    edit: () => string;
+    delete: () => string;
+    reset: () => string;
+  };
+  actionPopover: {
+    ariaLabel: () => string;
+  };
+  actionToolbar: {
+    selected: () => string;
+  };
+  batchSelection: {
+    selected: (count: number | string) => string;
+  };
+  confirm: {
+    no: () => string;
+    yes: () => string;
+  };
+  date: {
+    formats: {
+      inputs: () => string[];
+      javascript: () => string;
+    };
+  };
+  errors: {
+    messages: {
+      formSummary: (
+        errors: number | string,
+        warnings: number | string,
+        type: string
+      ) => [string, string] | null;
+    };
+  };
+  message: {
+    closeButtonAriaLabel: () => string;
+  };
+  numeralDate: {
+    validation: {
+      day: () => string;
+      month: () => string;
+      year: () => string;
+    };
+  };
+  pager: {
+    show: () => string;
+    records: (count: number | string, showNumber: boolean) => string;
```

Comment:
`count` can only be a string for the reason in my other comment

Reply:
https://github.com/Sage/carbon/blob/master/src/components/pager/pager.component.js#L203
![Zrzut ekranu 2021-09-22 o 10 19 05](https://user-images.githubusercontent.com/77274590/134308060-be857035-af7c-4d93-adef-f9bb1da55716.png)

I based on the prop types here. I think if we want to have only string type, we will have add toString():
https://github.com/Sage/carbon/blob/master/src/components/pager/pager.component.js#L167
and
https://github.com/Sage/carbon/blob/master/src/components/pager/pager.component.js#L159